From ca27d673838827848da1b975237781402ea8cf4d Mon Sep 17 00:00:00 2001
From: Wu Zhangjin <wuzhangjin@gmail.com>
Date: Mon, 6 Jan 2020 06:17:31 +0800
Subject: [PATCH] scripts: qemu: git-submodule.sh: update remote and checkout
 master

Signed-off-by: Wu Zhangjin <wuzhangjin@gmail.com>
---
 scripts/git-submodule.sh | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/scripts/git-submodule.sh b/scripts/git-submodule.sh
index 807ca0b..fe70884 100755
--- a/scripts/git-submodule.sh
+++ b/scripts/git-submodule.sh
@@ -71,11 +71,18 @@ update)
         exit 0
     fi
 
-    $GIT submodule update --init $modules 1>/dev/null
+    $GIT submodule update --init --remote $modules 1>/dev/null
     test $? -ne 0 && error "failed to update modules"
 
     $GIT submodule status $modules > "${substat}"
     test $? -ne 0 && error "failed to save git submodule status" >&2
+
+    for m in $maybe_modules
+    do
+        cd $m
+        git checkout master
+	 cd -
+    done
     ;;
 esac
 
-- 
2.7.4

