diff --git a/drivers/clk/clk.c b/drivers/clk/clk.c
index 96053a9..5615649 100644
--- a/drivers/clk/clk.c
+++ b/drivers/clk/clk.c
@@ -886,8 +886,7 @@ static int clk_core_enable(struct clk_core *core)
 	if (!core)
 		return 0;
 
-	if (WARN(core->prepare_count == 0,
-	    "Enabling unprepared %s\n", core->name))
+	if (core->prepare_count == 0)
 		return -ESHUTDOWN;
 
 	if (core->enable_count == 0) {
diff --git a/drivers/tty/serial/serial_core.c b/drivers/tty/serial/serial_core.c
index 351843f..b414235 100644
--- a/drivers/tty/serial/serial_core.c
+++ b/drivers/tty/serial/serial_core.c
@@ -467,7 +467,7 @@ static void uart_shutdown(struct tty_struct *tty, struct uart_state *state)
 		}
 	}
 	/* Should never happen */
-	WARN_ON(1);
+	// WARN_ON(1);
 	return 0;
 }
 
