From ef2794c82d2a7d3380a0663bb77c0c4c04293e47 Mon Sep 17 00:00:00 2001
From: Wu Zhangjin <wuzhangjin@gmail.com>
Date: Sun, 5 May 2019 12:51:47 -0700
Subject: [PATCH 2/2] qemu: capstone: fix up include path issue

Signed-off-by: Wu Zhangjin <wuzhangjin@gmail.com>
---
 include/disas/capstone.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/disas/capstone.h b/include/disas/capstone.h
index 84e2149..2eb6be2 100644
--- a/include/disas/capstone.h
+++ b/include/disas/capstone.h
@@ -3,7 +3,7 @@
 
 #ifdef CONFIG_CAPSTONE
 
-#include <capstone.h>
+#include <capstone/capstone.h>
 
 #else
 
-- 
2.7.4

