From 33afb6b11ef82724989e6dc14da6b507146c4d9e Mon Sep 17 00:00:00 2001
From: Wu Zhangjin <falcon@tinylab.org>
Date: Sun, 19 Jul 2020 17:30:57 +0800
Subject: [PATCH] use gitee mirror of qemu submodules

Signed-off-by: Wu Zhangjin <falcon@tinylab.org>
---
 .gitmodules | 30 +++++++++++++++---------------
 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index d108478..8434a6a 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,45 +1,45 @@
 [submodule "roms/seabios"]
 	path = roms/seabios
-	url = git://git.qemu-project.org/seabios.git/
+	url = https://gitee.com/tinylab/qemu-seabios.git/
 [submodule "roms/SLOF"]
 	path = roms/SLOF
-	url = git://git.qemu-project.org/SLOF.git
+	url = https://gitee.com/tinylab/qemu-SLOF.git
 [submodule "roms/ipxe"]
 	path = roms/ipxe
-	url = git://git.qemu-project.org/ipxe.git
+	url = https://gitee.com/tinylab/qemu-ipxe.git
 [submodule "roms/openbios"]
 	path = roms/openbios
-	url = git://git.qemu-project.org/openbios.git
+	url = https://gitee.com/tinylab/qemu-openbios.git
 [submodule "roms/openhackware"]
 	path = roms/openhackware
-	url = git://git.qemu-project.org/openhackware.git
+	url = https://gitee.com/tinylab/qemu-openhackware.git
 [submodule "roms/qemu-palcode"]
 	path = roms/qemu-palcode
-	url = git://git.qemu.org/qemu-palcode.git
+	url = https://gitee.com/tinylab/qemu-palcode.git
 [submodule "roms/sgabios"]
 	path = roms/sgabios
-	url = git://git.qemu-project.org/sgabios.git
+	url = https://gitee.com/tinylab/qemu-sgabios.git
 [submodule "dtc"]
 	path = dtc
-	url = git://git.qemu-project.org/dtc.git
+	url = https://gitee.com/tinylab/qemu-dtc.git
 [submodule "roms/u-boot"]
 	path = roms/u-boot
-	url = git://git.qemu-project.org/u-boot.git
+	url = https://gitee.com/tinylab/qemu-u-boot.git
 [submodule "roms/skiboot"]
 	path = roms/skiboot
-	url = git://git.qemu.org/skiboot.git
+	url = https://gitee.com/tinylab/qemu-skiboot.git
 [submodule "roms/QemuMacDrivers"]
 	path = roms/QemuMacDrivers
-	url = git://git.qemu.org/QemuMacDrivers.git
+	url = https://gitee.com/tinylab/qemu-QemuMacDrivers.git
 [submodule "ui/keycodemapdb"]
 	path = ui/keycodemapdb
-	url = git://git.qemu.org/keycodemapdb.git
+	url = https://gitee.com/tinylab/qemu-keycodemapdb.git
 [submodule "capstone"]
 	path = capstone
-	url = git://git.qemu.org/capstone.git
+	url = https://gitee.com/tinylab/qemu-capstone.git
 [submodule "roms/seabios-hppa"]
 	path = roms/seabios-hppa
-	url = git://github.com/hdeller/seabios-hppa.git
+	url = https://gitee.com/tinylab/qemu-seabios-hppa.git
 [submodule "roms/u-boot-sam460ex"]
 	path = roms/u-boot-sam460ex
-	url = git://git.qemu.org/u-boot-sam460ex.git
+	url = https://gitee.com/tinylab/qemu-u-boot-sam460ex.git
-- 
2.7.4

